keps/sig-storage/1979-object-storage-support/README.md

```diff
@@ -3,703 +3,738 @@
 
-### Static Provisioning
-"Driverless" allows the existing workloads to make use of COSI without the need for vendors to create drivers. The following steps show the details of the workflow:
-+ Admin creates `Bucket` instance that references an existing backend bucket.
-+ User creates a `BucketRequest` (BR) that references this `Bucket`.
-+ User creates `BucketAccessRequest` (BAR) that references their BR, a `BucketAccessClass`, and their secret containing access tokens.
-+ COSI detects the existence of the BAR and follows the [grant access](#grant-bucket-access) steps.
-+ COSI detects the existence of the BAR, BA and executes a validation process.
-+ As a validation step, COSI ensures that the `BucketRequest.bucketInstanceName` matches the `BucketAccessRequest.bucketAccessName.bucketInstanceName`. In other words, the two related `Bucket` references match, meaning that the BR and BAR->BA point to the same `Bucket`.
+The following stakeholders are involved in this lifecycle:
 
----
+ - Admin - creates the Bucket API object
+ - User - specifies bucket in their pod definition 
 
-# gRPC Definitions
+###### 1. Admin creates a Bucket API object
 
-There is one service defined by COSI - `Provisioner`. This will need to be satisfied by the vendor-provisioner in order to be COSI-compatible
+A Bucket with `bucketID` indicates that this Bucket has already been created outside of COSI. This bucket API object is not created as a result of a BucketRequest processed by COSI, it is rather created by the admin. 
 
 ```
-service Provisioner {
-    rpc ProvisionerGetInfo (ProvisionerGetInfoRequest) returns (ProvisionerGetInfoResponse) {}
+    Bucket - br-$uuid
+    |-------------------------------------------------|
+    | name: bucketName123                             |
+    | spec:                                           |
+    |   bucketID: bucketname123.s3.amazonaws.com      |
+    |   protocol: s3                                  |
+    |   parameters:                                   |
+    |     key: value                                  |
+    |   provisioner: s3.amazonaws.com                 | 
+    |-------------------------------------------------|
+```
 
-    rpc ProvisionerCreateBucket (ProvisionerCreateBucketRequest) returns (ProvisionerCreateBucketResponse) {}
-    rpc ProvisionerDeleteBucket (ProvisionerDeleteBucketRequest) returns (ProvisionerDeleteBucketResponse) {}
+###### 2. User requests BucketAccess to be created
 
-    rpc ProvisionerGrantBucketAccess (ProvisionerGrantBucketAccessRequest) returns (ProvisionerGrantBucketAccessResponse);
-    rpc ProvisionerRevokeBucketAccess (ProvisionerRevokeBucketAccessRequest) returns (ProvisionerRevokeBucketAccessResponse);
-}
+Unlike the BucketAccessRequest for COSI created bucket, this BucketAccess should directly reference the Bucket instead of a BucketRequest
```

Comment:
Typo. Should be "_Unlike the BucketAccessRequest for COSI created bucket, this BucketAccess**Request** should directly reference the Bucket instead of a BucketRequest_"